File: csharp/HelloWorld.cs
using System;

namespace Sample
{
    public class HelloWorld
    {
        private int num = 0;

        public HelloWorld()
        {
            num++;
        }

        public static int plus(int a,int b)
        {
            return a+b;
        }

        public int minus(int a,int b)
        {
            return a-b;
        }

        public void display()
        {
            Console.WriteLine("num={0}",num);
        }	
    }
}

File: java/HelloWorld.java
public class HelloWorld {
    int num = 0;

    public HelloWorld() {
        num+=1;
    }

    static int plus(int a, int b) {
        return a+b;
    }

    int minus(int a, int b) {
        return a-b;
    }

    void display() {
        System.out.printf("num=%d\n",num);
    }
}
